### ISSUE TITLE
change redshift_route_table_ids ?

### ISSUE THREAD

**Issue by officel** [terraform-aws-modules/terraform-aws-vpc#738]
## Description

The private route table assigned to the redshift subnet is changed to the public route table.

## Versions

```
Terraform v1.1.4
on linux_amd64
+ provider registry.terraform.io/hashicorp/aws v3.73.0
```

## Reproduction

I'm trying to use github actions to bring it up to date.
Today I noticed a diff.

### Code Snippet to Reproduce

no set var.enable_public_redshift, so default is false right?

## Expected behavior

no changes.

## Actual behavior

```
Changes to Outputs:
  ~ vpc = {
      ~ database_route_table_ids                    = [
          - "rtb-0dd99",
          - "rtb-01265",
          - "rtb-02445",
        ] -> [
          + "rtb-0dd99",
          + "rtb-01265",
          + "rtb-02445",
        ]
      ~ elasticache_route_table_ids                 = [
          - "rtb-0dd99",
          - "rtb-01265",
          - "rtb-02445",
        ] -> [
          + "rtb-0dd99",
          + "rtb-01265",
          + "rtb-02445",
        ]
      ~ redshift_route_table_ids                    = [
          - "rtb-0dd99",
          - "rtb-01265",
          - "rtb-02445",
        ] -> [
          + "rtb-0c6a7",
        ]
        # (106 unchanged elements hidden)
    }
```

## Additional context

Caused by this change
https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/729/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R253

Ignore the flag and use the public subnet.

**Comment by antonbabenko** [comment]
Thanks for opening this issue and providing a specific line that broke it. I will make a fix now.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.11.4 :tada:

**Comment by officel** [comment]
Thank you for your quick response and release!!
